spec/GraphQLOverHTTP.md

```diff
@@ -70,33 +90,70 @@ Note: This is an example of a non-normative note.
-but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Though [the GraphQL specification](https://spec.graphql.org) is transport
+agnostic, this GraphQL over HTTP specification aims to map GraphQL's semantics
+to their HTTP equivalents, enabling us to combine the full advantages of GraphQL
+with the rich feature set of HTTP.
+
+GraphQL queries and mutations naturally mirror the request/response message
+model used in HTTP, allowing us to provide a GraphQL request in an HTTP request
+and a GraphQL response in an HTTP response.
+
+:: In this document, the term {GraphQL schema} refers to a
+[schema as defined by the GraphQL specification](https://spec.graphql.org/draft/#sec-Schema).
+
+:: In this document, the term {GraphQL request} refers to a
+[request as defined by the GraphQL Specification](https://spec.graphql.org/draft/#request).
+
+:: In this document, the term {GraphQL response} refers to a
+[response as defined by the GraphQL Specification](https://spec.graphql.org/draft/#sec-Response).
+
+:: In this document, the term {GraphQL request error} refers to a
+[request error as defined by the GraphQL specification](https://spec.graphql.org/draft/#sec-Errors.Request-errors).
 
-GraphQL naturally follows the HTTP request/response message model,
-providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
+:: In this document, the term {GraphQL field error} refers to a
+[field error as defined by the GraphQL specification](https://spec.graphql.org/draft/#sec-Errors.Field-errors)
+
+:: In this document, the term {server} refers to a GraphQL over HTTP
+Specification compliant HTTP server unless the context indicates otherwise.
+
+:: In this document, the term {client} refers to a GraphQL over HTTP
+Specification compliant HTTP client unless the context indicates otherwise.
 
 # URL
 
-A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+A server MUST enable GraphQL requests to one of more GraphQL schemas.
+
+Each GraphQL schema a server provides MUST be served via one or more URLs.
 
-A GraphQL schema allows clients to know the available operations on a GraphQL server.
-Clients can discover the schema by sending an introspection query, which the server MUST answer with a proper response. 
+A client MUST be able to send all their GraphQL query and mutation requests for
+a single GraphQL schema made available by a server to a single URL endpoint on
+that server. A server MUST NOT require the client to use different URLs for
+different GraphQL query and mutation requests to the same GraphQL schema.
 
-The schema on a single URL MAY not be the same for every client.
-For example, certain fields could be restricted to authenticated users or alpha testers.
+Note: This means that a client must be able to issue all GraphQL query and
+mutation operations it needs via a single endpoint.
```

Comment:
Good question. The first two are required; imagine you have three endpoints for the same schema: `/a`, `/b` and `/c`.

Imagine you can issue all operations to `/a`, only named operations to `/b` and only unnamed operations to `/c`. This would pass sentence 1 but not sentence 2. But sentence 2 might cover both cases.

I think you're right; we should trim this down to just sentence 2.